packages/flutter_tools/lib/src/runner/flutter_command_runner.dart

```diff
@@ -220,6 +228,7 @@ class FlutterCommandRunner extends CommandRunner {
     // Set up the tooling configuration.
     if (enginePath != null) {
       ToolConfiguration.instance.engineSrcPath = enginePath;
```

Comment:
It's confusing that we have two distinct canonical places where we store the enginePath, the ToolConfiguration and the Command. Is there any reason not to combine these into one?